package/cpp/api/JsiSkPathFactory.h

```diff
@@ -165,7 +165,7 @@ class JsiSkPathFactory : public JsiSkHostObject {
                        JSI_EXPORT_FUNC(JsiSkPathFactory, MakeFromCmds),
                        JSI_EXPORT_FUNC(JsiSkPathFactory, MakeFromText))
 
-  JsiSkPathFactory(std::shared_ptr<RNSkPlatformContext> context)
+    JsiSkPathFactory(std::shared_ptr<RNSkPlatformContext> context)
```

Comment:
This commit should probably not be part of PR? Indentation looks wrong as well ;)